packages/host/app/services/matrix-service.ts

```diff
@@ -100,9 +104,14 @@ export default class MatrixService extends Service {
         Membership.onMembership(this),
       ],
       [this.matrixSDK.RoomEvent.Timeline, Timeline.onTimeline(this)],
+      [this.matrixSDK.UserEvent.DisplayName, this.onDisplayNameChange],
```

Comment:
probably we don't need this since profile name changes will only ever come from the host app of the currently logged in user...